spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkStagedScan.java

```diff
@@ -39,11 +38,6 @@
 
 public class TestSparkStagedScan extends CatalogTestBase {
 
-  public TestSparkStagedScan(
```

Comment:
this and the class below were running with the three different catalog configurations defined in `CatalogTestBase` and I've verified that this is still the case